modules/tinymce/src/plugins/emoticons/main/ts/api/Options.ts

```diff
@@ -40,7 +40,7 @@ const register = (editor: Editor, pluginUrl: string): void => {
 
   registerOption('emoticons_images_url', {
     processor: 'string',
-    default: 'https://twemoji.maxcdn.com/v/13.0.1/72x72/'
+    default: 'https://cdnjs.cloudflare.com/ajax/libs/twemoji/15.1.0/72x72/'
```

Comment:
I'm surprised that our product works like this. I mean this isn't really on-prem if you enable this it goes out on the internet for things.

Reply:
It is documented, though.
https://www.tiny.cloud/docs/tinymce/latest/emoticons/#emoticons_images_url

Speaking of which, we should change the docs URL for downloading the image assets to the new maintenance home as well... @ArvinJ-H can you please log a JIRA ticket for that

Reply:
Right I think this is the only thing that works like this that depends on a third party url. It's just a bit odd. I see why we do it we don't want to download that huge thing and ship that. But I guess most people just use the unicode emojis.

Reply:
> Speaking of which, we should change the docs URL for downloading the image assets to the new maintenance home as well... @ArvinJ-H can you please log a JIRA ticket for that

@ArvinJ-H Could you please create a documentation ticket if it hasn't been done yet?

Reply:
yes, there was a ticket logged to update the url in doc, forgot to link it in ticket.

https://ephocks.atlassian.net/browse/DOC-2406